glean-core/android/src/main/java/mozilla/telemetry/glean/private/BooleanMetricType.kt

```diff
@@ -27,7 +27,8 @@ class BooleanMetricType(
                 name = name,
                 send_in_pings = ffiPingsList,
                 send_in_pings_len = sendInPings.size,
-                lifetime = lifetime.ordinal)
+                lifetime = lifetime.ordinal,
+                disabled = if (disabled) { 1 } else { 0 })
```

Comment:
What if we write an extension function for this, in `LibGleanFFI.kt`? We could do something like:

```
// Comment about what this does and why we need it
internal fun Boolean.toInt() = if (this) 1 else 0
```

This way the code above can be reduced to:

`disabled = disabled.toInt()`